client/game/SocialExposure.jsx

```diff
@@ -0,0 +1,42 @@
+import React from "react";
+import Slider from "meteor/empirica:slider";
+
+export default class SocialExposure extends React.Component {
+  renderSocialInteraction(otherPlayer) {
+    const value = otherPlayer.round.get("value");
+    return (
+      <div className="alter" key={otherPlayer._id}>
+        <img src={otherPlayer.get("avatar")} className="profile-avatar" />
+        Guess: {value}
+      </div>
+    );
+  }
+
+  render() {
+    const { game, player } = this.props;
+
+    const otherPlayers = _.filter(
+      game.players,
+      p =>
+        p._id != player._id && player.get("neighbors").includes(p.get("nodeId"))
```

Comment:
This arrow function might be kinda hard to parse for a beginner. I think "neighbors" should not contain the current user from the get go. Maybe also we can pull out `player.get("neighbors")` into a var above `const otherPlayers`

Would give something like:

```suggestion
        neighbors.includes(otherPlayer.get("nodeId"))
```

Reply:
agreed re: neighbors doesn't contain player, this is some weird Frankenstein leftovers from when i copied abdullah's tutorial code and turned it into network code

agreed re: arrow

Reply:
Yeah, this one actually changes completely with moving neighbors to player IDs, my bad for the not updating this comment after I made that proposal below.